packages/service/src/server/middlewares/getAssetMiddleware.ts

```diff
@@ -1,12 +1,20 @@
+import { SERVER_REQUEST } from '@shuvi/shared/constants/trace';
 import { IServerPluginContext } from '../plugin';
 import { ShuviRequestHandler } from '../../server';
 import { CLIENT_OUTPUT_DIR } from '../../constants';
 import { isStaticFileExist, serveStatic } from '../utils';
 
+const { SHUVI_SERVER_HANDLE_REQUEST_START, SHUVI_SERVER_RUN_ASSET_MIDDLEWARE } =
+  SERVER_REQUEST.events;
+
 export const getAssetMiddleware = (
   context: IServerPluginContext
 ): ShuviRequestHandler => {
   return async (req, res, next) => {
+    const { serverRequestTrace } = context.traces;
+    serverRequestTrace
+      .traceChild(SHUVI_SERVER_HANDLE_REQUEST_START.name)
```

Comment:
SHUVI_SERVER_HANDLE_REQUEST_START has been moved here because assisMiddleware is the first middleware